qt/TorrentDelegateMin.cc

```diff
@@ -49,8 +49,8 @@ namespace
 class ItemLayout
 {
 private:
-    QString myNameText;
-    QString myStatusText;
+    QString name_text_;
+    QString status_text_;
 
 public:
     QFont nameFont;
```

Comment:
Do public fields get the same treatment or not? Maybe there shouldn't be any public fields?

Reply:
I'd say yes, all fields should get the same treatment.

I wouldn't hold it as a _strict_ rule, but yeah as a guideline we should limit public / protected fields -- that's far out-of-scope for this PR though